python/pyspark/sql/connect/client/core.py

```diff
@@ -1057,6 +1057,24 @@ def execute_command_as_iterator(
                     },
                 )
 
+    def execute_ml(self, req: pb2.ExecutePlanRequest):
+        """
+        Execute the ML command request and return ML response result
+        Parameters
+        ----------
+        req : pb2.ExecutePlanRequest
+            Proto representation of the plan.
+        """
+        logger.info("Execute ML")
+        try:
+            for attempt in self._retrying():
+                with attempt:
+                    for b in self._stub.ExecutePlan(req, metadata=self._builder.metadata()):
```

Comment:
This one will need some additional work to properly deal with the reattach behavior. It might be better to reuse the existing response iterator behavior, but that is not too complicated.

Reply:
Ok, I will check how to use it. Thx for the suggestion.